### ISSUE TITLE
TypeError: object of type 'NoneType' has no len()

### ISSUE THREAD

**Issue by bralbral** [ccxt/ccxt#15941]
- OS: Debian GNU/Linux 11 (bullseye)
- Programming Language version: python 3.10.0 (default, Nov  3 2022, 19:19:49) [GCC 10.2.1 20210110]
- CCXT version:  2.2.63

```
    # located at .../ccxt/async_support/base/exchange.py
    def parse_ohlcvs(self, ohlcvs, market=None, timeframe='1m', since=None, limit=None):
        results = []
        for i in range(0, len(ohlcvs)):
            results.append(self.parse_ohlcv(ohlcvs[i], market))
        sorted = self.sort_by(results, 0)
        tail = (since is None)
        return self.filter_by_since_limit(sorted, since, limit, 0, tail)
```
Code from [ccxt](https://github.com/ccxt/ccxt/blob/master/python/ccxt/async_support/base/exchange.py#L1120)
```
2022-12-03 19:26:46,217 - root - ERROR - Traceback (most recent call last):
  File "/home/bral/PycharmProjects/PoloniexTradingBot/fetcher.py", line 82, in fetch
    async for candles in iterator:
  File "/home/bral/PycharmProjects/PoloniexTradingBot/core/utils.py", line 125, in fetch_ohlcv_async_iterator
    candles = await client.fetch_ohlcv(
  File "/home/bral/PycharmProjects/PoloniexTradingBot/venv/lib/python3.10/site-packages/ccxt/async_support/bybit.py", line 2043, in fetch_ohlcv
    return await self.fetch_derivatives_ohlcv(symbol, timeframe, since, limit, params)
  File "/home/bral/PycharmProjects/PoloniexTradingBot/venv/lib/python3.10/site-packages/ccxt/async_support/bybit.py", line 2026, in fetch_derivatives_ohlcv
    return self.parse_ohlcvs(ohlcvs, market, timeframe, since, limit)
  File "/home/bral/PycharmProjects/PoloniexTradingBot/venv/lib/python3.10/site-packages/ccxt/async_support/base/exchange.py", line 1120, in parse_ohlcvs
    for i in range(0, len(ohlcvs)):
TypeError: object of type 'NoneType' has no len()

```
Perhaps you need to do a check, for example: `isinstance(ohlcvs,list)` ?

**Comment by carlosmiei** [comment]
Hello @bralbral, Can you share some more details that could help us debug the issue? Things like `symbol` used and a verbose log.

**Comment by bralbral** [comment]
Hello @carlosmiei 

Just minimal reproducible example for you:
```

from ccxt.async_support.bybit import bybit
import asyncio


async def main() -> None:
    # Saturday, 4 December 2021, 17:00:33
    since = 1638637233000
    client = bybit()
    ohlcv = await client.fetch_ohlcv(symbol="LTCUSD", since=since)
    print(ohlcv)


if __name__ == "__main__":
    asyncio.run(main())
```

**Comment by bralbral** [comment]
seems fixed in 2.4.17
